DTechnology/settings.py

```diff
@@ -27,7 +28,7 @@
 SECRET_KEY = config('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Is debug mode enabled in production? I think this should be changed, if static files does not work it could be useful https://docs.djangoproject.com/en/4.1/howto/static-files/deployment/

Reply:
I know, but I prefer to change that later when the project is finished, to avoid conflicts.